config/params/mainnet_config.go

```diff
@@ -17,7 +18,7 @@ func MainnetConfig() *BeaconChainConfig {
 
 // UseMainnetConfig for beacon chain services.
 func UseMainnetConfig() {
-	beaconConfig = MainnetConfig()
+	OverrideBeaconConfig(MainnetConfig().Copy())
```

Comment:
here too , `OverrideBeaconConfig` is used in non testing code.

Reply:
Good point, I will run through all the `Use<ConfigName>` methods and make sure they aren't  used outside of tests. I may remove them to make it easier to scan for `OverrideBeaconConfig`.

Reply:
I killed all the `Use<x>` methods and did a thorough search for things outside of tests using `OverrideBeaconConfig`. Please double check my work if you can - this command was my todo list:
```
grep -r OverrideBeaconConfig | grep -v '_test.go' | grep -v 'testing/spectest' | grep -v \.git
```